### ISSUE TITLE
Not able to XC any frequency.

### ISSUE THREAD

**Issue by ImAlex28** [pierr3/TrackAudio#246]
**Issue Summary:** Unable to select to XC a frequency from the UI by right clicking XCA; the XCA button remains unchanged. However, XCA left click functionality works without issues.

**Steps to Reproduce:**
1. Right click XCA button of a FREQ while connected as ATC.
2. Observe that the button does not change, XC is not applied neither.

**Expected Behavior:** The button should change when selecting XC, and the transceivers of that FREQ should be coupled.

**Actual Behavior:** The button remains the same when attempting to select XC and transceivers are not coupled.

**Comment by GeorgeBarlow** [comment]
The issue seems to be with how we are setting XCA or XC on the C++ side.

https://github.com/pierr3/TrackAudio/blob/18e24fd3b404fcd02313fdfc33f6e95f066e555c/backend/include/RadioHelper.hpp#L49-L57

https://github.com/pierr3/afv-native/blob/def3739684b8b0e66f5023cc582ab35ebdbef51c/src/afv/ATCRadioSimulation.cpp#L817-L839

The faulty line seems to be https://github.com/pierr3/afv-native/blob/def3739684b8b0e66f5023cc582ab35ebdbef51c/src/afv/ATCRadioSimulation.cpp#L833

Since when we click `XC` and trigger `SetRadioState` in RadioHelper and calling both 
```
mClient->SetXc(newState.frequency, newState.xc);
mClient->SetCrossCoupleAcross(newState.frequency, newState.xca);
```
We run into an issue where when we click `XC` and subsequently set it to true on the radio state inside afv-native, using `SetXc`, we then are setting it back to false as we call `SetCrossCoupleAcross` we use the value of `XCA` as the parameter `crossCoupleAcross` and XCA, of course, has to be false when setting `XC` to true. Therefore the line `mRadioState[freq].xc = crossCoupleAcross;` inside `SetCrossCoupleAcross` reverts the `XC` property back to false. 

TLDR: @pierr3 can we get rid of `mRadioState[freq].xc = crossCoupleAcross;` inside `SetCrossCoupleAcross`

**Comment by pierr3** [comment]
Did not manage to get this fixed in this release, but it will be in the next!

**Comment by pierr3** [comment]
Nevermind, this is fixed in the upcoming beta 5
